### ISSUE TITLE
JBoss scanning should try to use the locate command

### ISSUE THREAD

**Issue by chambridge** [quipucords/rho#291]
## Specify type:
- Enhancement
___

## Description:
When the Jboss scanner tasks that use a 'find' command run on large filesystems the commands take a long time to complete (sometimes days) causing the script to appear hung.

Related to #273

___

### Version of rho:
ALL

### Environment information:
Target filesytem was NFS and below is the size and path.

> ipxnas03.midwestiso.org:/sysdeploy
>                       6.9T  5.2T  1.8T  75% /sysdeploy

___

Instead of using the `find` command we should attempt to use the `locate` command (if installed). 

If the `locate` command is not installed we will fall back to utilizing the `find` command.

**Comment by chambridge** [comment]
@mdvickst Did I capture what you were looking for in the above enhancement?

**Comment by mdvickst** [comment]
I think if the 'locate' command is present we should use that first to reduce load on the server being scanned. 

I don't know enough about locate to know where it scans by default so one potential obstacle to using it might be if other filesystems are not included by default (I think NFS is not indexed by locate by default).

**Comment by noahl** [comment]
I found two implementations of locate: the one in GNU findutils, and mlocate, which is a replacement with more efficient database updates.

We'll have to be sure to test with both.

**Comment by noahl** [comment]
Oh, and slocate is also a thing.

**Comment by noahl** [comment]
As far as I can tell, plain findutils locate was removed from Fedora/RHEL (Red Hat linux?) in 2000, so testing that is probably a low priority.

I found package lists for RHEL 6 and 7 and I don't see slocate in there. I suspect it's not in RHEL 5 either, but that's based on some forum post I found and not on a real source. However, I feel safe saying that the vast majority of our (RHEL) users will have mlocate.
